proto/agglayer/interop/types/v1/aggchain.proto

```diff
@@ -8,10 +8,40 @@ import "agglayer/interop/types/v1/bytes.proto";
 message AggchainData {
   // The data of the aggchain.
   oneof data {
-    // Signature committed to the bridge exits and imported bridge exits.
+    // Legacy ECDSA signature on a commitment to the state transition.
     FixedBytes65 signature = 1;
-    // Generic Aggchain proof.
+
+    // Aggchain proof with single ECDSA verified by the agglayer only.
```

Comment:
```suggestion
    // Aggchain proof (including full execution proof) for the network state transition signed with the single ECDSA (signature is verified by the agglayer only).
```

Reply:
I wont add `(including full execution proof)` because that's not the case, that's the aggchain proof (which indeed verifies an FEP inside, but the API here doesnt care about that)

so I'm ok with this suggestion
```suggestion
    // Aggchain proof with the single ECDSA verified by the agglayer only.
```